### PR TITLE
[10.0][MIG] Migration of account_analytic_project to version 10.0

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

**Review by ThomasBinsfeld — APPROVED** [review]

File: account_analytic_project/model/account_analytic_account.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+class AccountAnalyticAccount(models.Model):
+    """Extend account.analytic.account."""
+    _inherit = 'account.analytic.account'
+
+    @api.multi
+    def create_project_for_account(self):
+        """Create a project for an already existing analytic account."""
+        project_model = self.env['project.project']
+        for this in self:
+            if not this.project_ids and this.type != 'view':
```
Comment (ThomasBinsfeld) [inline account_analytic_project/model/account_analytic_account.py]: @adrienpeiffer `type` field does not exist on `account.analytic.account` that's why Travis is red.

**Review by ThomasBinsfeld — CHANGES_REQUESTED** [review]

**Comment by adrienpeiffer** [comment]
The possiblity to access on projects from analytic account is now in Odoo Core (and the relational field too). So, it's remains only the improvement to create a project from existing analytic accounts.

File: account_analytic_project/model/account_analytic_account.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    @api.multi
+    def create_project_for_account(self):
+        """Create a project for an already existing analytic account."""
+        project_model = self.env['project.project']
+        for this in self:
+            if not this.project_ids:
+                project_model.create({
+                    'name': this.name,
+                    'analytic_account_id': this.id,
+                })
```
Comment (ThomasBinsfeld) [inline account_analytic_project/model/account_analytic_account.py:20]: @adrienpeiffer Maybe a prepare method for this ?

**Review by ThomasBinsfeld — APPROVED** [review]
Just an improvement suggestion. Otherwise LGTM (functional and code review) :+1:

**Pull request closed without merging**